libvcx/readme.md

```diff
@@ -3,10 +3,8 @@ You can filter out tests by specifying features they require / use.
 ```
 - general_test :: does not require any external component
 - pool_tests   :: requires indypool to be running
-- agency_v2    :: requires agency talking libvcx client2agency v2 protocol (nodevcx-agency)
+- agency_tests    :: requires agency talking libvcx client2agency v2 protocol (nodevcx-agency)
```

Comment:
"requires a cloud agency speaking libvcx client-2-agency v2+ protocol (e.g. vcxagencynode)" - `vcxagencynode` is the name of the repo, `vcxagency-node` is the name of the npm [package](https://github.com/AbsaOSS/vcxagencynode/blob/master/vcxagency-node/package.json#L2). Our [readme](https://github.com/hyperledger/aries-vcx) calls it `NodeVCX Agency`. The docker image is called `vcxagency-node`. I think I was pushing toward `node-vcx-agency` to make the naming analogous to `node-vcx-wrapper` and many other node projects. I guess this thing trademarks all permutations of the words node, vcx, and agency with or without dashes and all possible capitalizations 😄 

Also, please, allign the `::`.